packages/vats/src/core/chain-behaviors.js

```diff
@@ -146,9 +147,24 @@ export const bridgeProvisioner = async ({
     async fromBridge(_srcID, obj) {
       switch (obj.type) {
         case 'PLEASE_PROVISION': {
-          const { nickname, address, powerFlags } = obj;
-          return E(provisioning)
-            .pleaseProvision(nickname, address, powerFlags)
+          const { nickname, address, powerFlags: rawPowerFlags } = obj;
+          const powerFlags = rawPowerFlags || [];
```

Comment:
don't we usually write it this way?

```suggestion
          const { nickname, address, powerFlags = [] } = obj;
```

Reply:
- wish: unit test
 - wish: pattern-based dynamic type check on `obj`

Reply:
Turns out that `obj.powerFlags` can be `null`.  The destructuring default values only work if `obj.powerFlags` is `undefined`.

Yeah, the dynamic checks would be better with patterns.